bylaws/0-preface.md

```diff
@@ -0,0 +1,63 @@
+---
+status: draft
+weight: 0
+---
+# Preface
+
+Welcome to the Commonhaus Foundation (CF), a collective dedicated to nurturing open-source projects through collaboration and innovation. These bylaws provide a detailed guide to our governance structure and outline your role within our community.
+
+## Key Terms
+
+- **[CF Council (CFC)][cfc]**: Governs CF operations and strategic direction.
+- **[CF Advisory Board][cfab]**: Comprises representatives from organizations providing financial support to the CF. They provide strategic insight to the CFC.
+- **[General Members][membership]**: Individuals supporting CF’s mission. Members in good standing participate in CF activities and vote in elections.
+- **[Projects][project]**: Collaborative open-source initiatives under CF, including assets like code, trademarks, and domains.
+  - **[Code Owners][plco]**: Maintain the quality and consistency of project code.
+  - **[Project Leaders][plco]**: Guide project governance and decision-making. Along with the CFC, they form the [**Extended Governance Committee (EGC)**][egc] for critical decisions like bylaw amendments.
+
+## Decision Making and Voting
+
+Our [decision-making process][cf-decision-making], based on Lazy Consensus, values every member’s voice, ensuring equality and empowerment.
+
+## Record Keeping
+
+We commit to thorough and accessible documentation of CF activities, detailed in [Record Keeping][records].
+
+## Governance Policies
+
+In addition to our bylaws, the Commonhaus Foundation (CF) adopts specific policies to address focused areas of our operations and community interactions.
+
+- [**Code of Conduct**][policy-coc]: Definition of behavioral expectations and conflict resolution and remediation in support of a respectful and inclusive community.
+- [**Conflict of Interest**][policy-coi]: Establishes protocols for identifying, disclosing, and managing any personal or financial interests that could potentially interfere with the impartial and objective operation of the CF.
+- [**Intellectual Property Policy**][policy-ip]: Framework for responsibly managing and protecting intellectual contributions.
+- [**Succession Plan**][policy-sp]: Strategy to ensure leadership continuity and project resilience.
+- [**Trademark Policy**][policy-tp]: Guidelines for the appropriate use of CF’s intellectual property.
+
+See the [index of CF policies][policy-index] for a complete list.
+
+## Bylaw change process
+
+The [foundation][] repository is the authoritative source for Commonhaus Foundation (CF) Bylaws.
+
+The CF Council (CFC) must review bylaws at least annually to ensure ongoing compliance with applicable laws and regulations.
+
+The [Extended Governance Committee (EGC)][egc] (EGC: CFC + Project Leads) jointly review and approve bylaw changes. For details on amending the bylaws, refer to the amendment process.
+
+Amendments or changes to Bylaws will follow the [amendment process][].
```

Comment:
Might be on purpose, but just in case it's an oversight: missing link here.

Reply:
Naaaa, these are perfectly fine implicit markdown links.

Reply:
Hmm, okay. I wasn't familiar with this syntax.